### ISSUE TITLE
Cannot be run on another platform than OS X

### ISSUE THREAD

**Issue by pepri** [LinusU/node-ds-store#3]
The dependency `macos-alias` requires platform OS X (`os: darwin`). This means I cannot install `ds-store` package in my environment.

I would like to create `.DS_Store` file in a build script that is run on Linux build server. Is the platform limitation really required? It would be nice if this was platform independent.

**Comment by LinusU** [comment]
Yeah, I would love to have it platform independent. Patches welcome!

**Comment by pepri** [comment]
Yep, I am looking into it. I will have OS X later today available, so I will make a patch. The patch will be actually made to `macos-alias`.

**Comment by chibizor** [comment]
Whatever happened to this? I am using Linux and badly need to create a DS_Store file without requiring a Mac. If node-ds-store could achieve this then it would solve a problem many other projects have been unable to conquer. You'll be heroes!

**Comment by pepri** [comment]
Yeah, I did not finish it completely. Changed the job, lost the source code and I do not have a use case for this anymore. Sorry for the bad news.
